e2e-tests/e2e-legacy/resource/resource.spec.js

```diff
@@ -370,7 +366,8 @@ describe.skip('Resource view', () => {
         });
     });
 
-    context('Download as', () => {
+    // TODO: Fix me. Broken NOT due to migration (Error: CORS violation when downloading from https://w3c.github.io/json-ld-api/tests/compact/0007-context.jsonld)
```

Comment:
Interesting. This should not be the case.

Reply:
See this comment https://github.com/Ontotext-AD/graphdb-workbench/blob/GDB-10373-workbench-with-single-spa/e2e-tests/e2e-flaky/import/import-server-files-operations.spec.js#L64
I'm pretty sure that this download test relies on graphdb to be started with the whitelist for this resource